### DISCUSSION TITLE
Live Stream Mode

### DISCUSSION THREAD

**Discussion by chasemsutton** [ThioJoe/YT-Spammer-Purge#649] [Ideas]
Is there any way you could make a mode that you link to your live stream that scans the live chat every 5 seconds or so and does the same filtering and deleting you have set up for video comments? Just don’t have enough mods on the channel to keep chat clean and free of scams.

**Comment by Firecul** [comment]
This has come up in #339 and #543 before, at last reply, ThioJoe said [he'd be looking into working with live stream chats,](https://github.com/ThioJoe/YT-Spammer-Purge/discussions/543#discussioncomment-2115068) not necessarily working on a timer like you suggest though. I'd guess that could eat up the quota pretty quick but I don't know the quota for those calls so might be feasible. Certainly you could make a config file for it to run automatically when opened then make task scheduler run it for example.

**Reply by chasemsutton** [reply]
Ya my initial plan was the task scheduler method until I realized that the app didn’t have live chat support. At that point I figured I’d ask for it all built into a nice little live stream mode. Anyway, thanks for pointing out that he’s working on it that gives me hope!
